inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp

```diff
@@ -1427,7 +1427,7 @@ void MKLDNNReduceNode::initSupportedPrimitiveDescriptors() {
     Precision inputPrecision = getOriginalInputPrecisionAtPort(REDUCE_DATA);
     Precision outputPrecision = getOriginalOutputPrecisionAtPort(0);
 
-    jit_mode = (mayiuse(cpu::x64::sse41)) && getParentEdgeAt(REDUCE_DATA)->getDims().ndims() <= 5 &&
```

Comment:
What if the machine supports sse41 while not supporting avx2? This case jit_mode gets false. sse41 can't be utilized. 
I think the original code here is right. Please also refer to the following comment: https://github.com/openvinotoolkit/openvino/pull/580#discussion_r457555148
Thanks Yury!

Reply:
Thanks, @xuchen-intel, I agree, that such solution seems to be a workaround, but we still have to fix original bug in sse4.1 case. I found it locates in reduce_main() function, but still cannot found the separate 'illegal instruction'.  May be, it doesn't connected with specific operator itself. Have you any ideas?

Reply:
I couldn't reproduce this problem with CPU-test. It passes on no-AVX2 machine.

Reply:
Thanks Yury! If you've already been working on the JIT details, maybe you can try to reproduce this problem by changing the following code. https://github.com/openvinotoolkit/openvino/blob/76131da7ca940532da3625e59250b98ce69ff7bf/inference-engine/src/mkldnn_plugin/nodes/mkldnn_reduce_node.cpp#L1528
Remove the condition branch of avx512 and avx2, so that it will be forced to utilize sse4.2 instruction.
Anyway, I'm OK with the workaround. And you can always to assign the issue to me, if you come across any problem.

Reply:
I tried to remove those conditions. It doesn't helps, so I have had to reproduce the problem with no-AVX machine via benchmark_app and GNMT model. Solution with additional `(mayiuse(cpu::x64::sse41))` works good. I tried to reproduce it in special test for reduce_node to reassign it reasonably, but without result.

Reply:
Anyway, thanks for advice. I'll consult with our team about acceptability of workaround in given case.